src/module/item/data-model-ability.js

```diff
@@ -21,15 +23,15 @@ export default class OseDataModelAbility extends foundry.abstract.DataModel {
   get #rollTag() {
     if (!this.roll) return null;
 
-    const rollTarget =
-      this.rollTarget === undefined
-        ? ""
-        : ` ${CONFIG.OSE.roll_type[this.rollType]}${this.rollTarget}`;
-
     return {
-      label: `${game.i18n.localize("OSE.items.Roll")} ${
-        this.roll
-      }${rollTarget}`,
+      label: `${game.i18n.localize("OSE.items.Roll")} ${OseTags.rollTagFormula({
```

Comment:
Last thing, promise. For legibility, would you mind moving the separate parts of this string (roll label, roll formula, roll target) out into `const`s in this getter?

Then we'd get something like 
```ts
{
  label: `${rollLabel} ${rollFormula}${rollTarget}`
}
```